sdk/textanalytics/ai-text-analytics/test/public/textAnalyticsClient.spec.ts

```diff
@@ -58,1499 +58,1508 @@ describe("[AAD] TextAnalyticsClient", function() {
+          {
+            entityRecognitionTasks: [{ modelVersion: "latest" }],
+            entityRecognitionPiiTasks: [{ modelVersion: "latest" }],
+            keyPhraseExtractionTasks: [{ modelVersion: "latest" }]
+          },
+          {
+            polling: {
+              updateIntervalInMs: pollingInterval
+            }
+          }
+        );
+        const result = await poller.pollUntilDone();
+        for await (const page of result) {
+          const entitiesResult = page.entitiesRecognitionResults!;
+          assert.equal(entitiesResult.length, 1);
+          for (const entitiesDocs of entitiesResult) {
+            assert.equal(entitiesDocs.length, 3);
+            for (const doc of entitiesDocs) {
+              assert.isUndefined(doc.error);
+            }
           }
         }
-      );
-      const result = await poller.pollUntilDone();
-      let docCount = 0;
-      let pageCount = 0;
-      const pageSize = 10;
-      for await (const page of result.byPage({ maxPageSize: pageSize })) {
-        const entitiesTaskDocs = page.entitiesRecognitionResults![0];
-        ++pageCount;
-        for (const doc of entitiesTaskDocs) {
-          assert.isUndefined(doc.error);
-          ++docCount;
-          if (!doc.error) {
-            if (docCount === totalDocs) {
-              assert.equal(doc.entities.length, 3);
-            } else {
-              assert.equal(doc.entities.length, 0);
+      });
+
+      it("invalid language hint", async function() {
+        const docs = ["This should fail because we're passing in an invalid language hint"];
+
+        const poller = await client.beginAnalyze(
+          docs,
+          {
+            entityRecognitionTasks: [{ modelVersion: "latest" }],
+            entityRecognitionPiiTasks: [{ modelVersion: "latest" }],
+            keyPhraseExtractionTasks: [{ modelVersion: "latest" }]
+          },
+          "notalanguage",
+          {
+            polling: {
+              updateIntervalInMs: pollingInterval
             }
           }
+        );
+        const result = await poller.pollUntilDone();
+        const firstResult = (await result.next()).value;
+        const entitiesTaskDocs = firstResult?.entitiesRecognitionResults![0];
+        for (const doc of entitiesTaskDocs) {
+          assert.equal(doc.error?.code, "UnsupportedLanguageCode");
         }
-      }
-      assert.equal(docs.length, docCount);
-      assert.equal(Math.ceil(docs.length / pageSize), pageCount);
-    });
+        const piiEntitiesTaskDocs = firstResult?.piiEntitiesRecognitionResults![0];
+        for (const doc of piiEntitiesTaskDocs) {
+          assert.equal(doc.error?.code, "UnsupportedLanguageCode");
+        }
+        const keyPhrasesTaskDocs = firstResult?.keyPhrasesExtractionResults![0];
+        for (const doc of keyPhrasesTaskDocs) {
+          assert.equal(doc.error?.code, "UnsupportedLanguageCode");
+        }
+      });
+
+      it.skip("bad model", async function() {
```

Comment:
why skip?

Reply:
this is not a new change by this PR. Anyway, it was an issue in the service that we hope it will be fixed in the near future.